src/excel_adapter/excel_adapter.go

```diff
@@ -15,29 +15,20 @@ type ExcelAdapterConfig struct {
 	Parse    internals.ParseConfig
 }
 
-type ExcelAdapter struct {
-	config     ExcelAdapterConfig
-	boards     map[string]models.Board
-	globalInfo models.GlobalInfo
-	document   internalModels.Document
-}
-
-func NewExcelAdapter(config ExcelAdapterConfig) ExcelAdapter {
-	//TODO: poner config del download nesteado en el TOML
+func FetchBoardsAndGlobalInfo(config ExcelAdapterConfig) (map[string]models.Board, models.GlobalInfo) {
```

Comment:
just to know your opinion, what about splitting this method into two different ones and then calling both in main?

Reply:
I will do it in the next PR, which introduces some changes to the excelAdapter